### ISSUE TITLE
Should merge policies defined in the config with the policies given to the action creator

### ISSUE THREAD

**Issue by viniciusdacal** [viniciusdacal/redux-arc#6]

**Comment by viniciusdacal** [comment]
A good solution for this, is adding the config props to our creators and let the middlewares we pass in the moment we are dispatching the action, overwrite the original ones. So, this way we could ignore the original middlewares, by doing:

```js
dispatch(creators.read({
  middlewares: ['useOnlyThisMiddleware'],
));
```

Or we could keep the config middlewares, by doing:

```js
dispatch(creators.read({
  middlewares: [...creators.read.middlewares, 'useOnlyThisMiddleware'],
));
```

That will also allow us to have a better debugging. Since we could console the creators and check if the config we set is right.

**Comment by viniciusdacal** [comment]
Implemented in current release 0.6.0
